### ISSUE TITLE
Bolt 2.2.0/1 does not work from subdirectory [regression]

### ISSUE THREAD

**Issue by awrog** [bolt/bolt#3647]
Installed a fresh installation Bolt 2.2.0 (also 2,.2.1) in www.arnorog.nl/bolt220
Let it prefill some data
Default menu contains:
main:
    - label: Home
      title: This is the first menu item.
      path: homepage
      class: first
    - label: Second item
      path: entry/1
      submenu:
          - label: Sub 1
            path: entry/2
          - label: Sub 2
            class: menu-item-class
            path: entry/3
          - label: Sub 3
            path: entry/4
          - label: Sub 4
            class: sub-class
            path: page/3
    - label: All pages
      path: pages/
    - label: The Bolt site
      link: http://bolt.cm
      class: last

The resulting links from the menu direct to the root of the webserver, not the installation directory. They don't work

Example
The first entry (Sub 1) under menu 'Second item' points to www.arnorog.nl/entry/2
It should point to www.arnorog.nl/bolt220/entry/2
This is the case for all entries in the menu.

This problem did not occur in Bolt 2.1.8

**Comment by GawainLynch** [comment]
Ouch… Yeah, I am guessing this is the new menu class.

Also, I can't thank you enough for actually giving us a live site to see the problem on… Seriously awesome! :100:

**Comment by bobdenotter** [comment]
I can reproduce this locally as well! Thanks, we're going to fix it.

**Comment by GawainLynch** [comment]
@awrog there is a PR fix just submitted. But for now a workaround is a one line fix.

In `src/Helpers/MenuBuilder.php` on line 137 change:
```php
        $item['link'] = '/' . $path;
```
to:
```php
        $item['link'] = $this->app['resources']->getUrl('root') . $path;
```

**Comment by bobdenotter** [comment]
Fixed by #3649 / #3648
